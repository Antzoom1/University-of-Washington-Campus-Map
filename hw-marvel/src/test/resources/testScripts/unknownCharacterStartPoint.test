# Test to find a path from an unknown character to a character in staffSuperheroes

LoadGraph staffSuperheroes staffSuperheroes.csv

FindPath staffSuperheroes OnePunchMan Notkin-of-the-Superhuman-Beard

