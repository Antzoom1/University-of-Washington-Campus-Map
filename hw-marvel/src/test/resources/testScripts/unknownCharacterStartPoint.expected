# Test to find a path from an unknown character to a character in staffSuperheroes

loaded graph staffSuperheroes

unknown: OnePunchMan
